src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -438,7 +441,7 @@ private static void FixNuSpecIdCasing(string nuspecFile, string targetNuspec, st
             PackageReaderBase packageReader,
             PackagePathResolver packagePathResolver,
             PackageSaveMode packageSaveMode,
-            XmlDocFileSaveMode xmlDocFileSaveMode,
+            PackageExtractionContext packageExtractionContext,
```

Comment:
done

---
In reply to: [59911413](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r59911413) [](ancestors = 59911413)